src/dbcp/transform/acp_projects.py

```diff
@@ -0,0 +1,458 @@
+"""Transform and clean ACP projects data."""
+import hashlib
+import re
+from functools import partial
+from typing import Callable
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+
+from dbcp.extract.fips_tables import CENSUS_URI, _extract_census_counties
+from dbcp.transform.helpers import add_county_fips_with_backup_geocoding
+
+
+def _rename_columns(raw_cols: pd.Index) -> pd.Index:
+    """Rename columns to snake case."""
+
+    def _camel_to_snake(name: str) -> str:
```

Comment:
Maybe add this to a helper module?

Reply:
I think the `_camel_to_snake` function is not quite generalized enough to to use elsewhere. It currently doesn't handle the (common) case of multiple consecutive capital letters, such as from abbreviations eg. `EIAPlantCode` (should output `eia_plant_code` but currently outputs `eiaplant_code`).